src/main/java/com/tarento/assetManagement/controller/AssetController.java

```diff
@@ -0,0 +1,41 @@
+package com.tarento.assetManagement.controller;
+
+import com.tarento.assetManagement.dao.dto.Asset;
+import com.tarento.assetManagement.service.AssetService;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/admin")
```

Comment:
let's change /admin to /org/user 
/org/admin will be used for  admin related task.

Reply:
Sure Manzarul, will make changes